spec/characterization_services/external_metadata_characterization_service_spec.rb

```diff
@@ -42,7 +42,7 @@
   end
 
   context "with an non-geo metadata file" do
-    let(:file) { fixture_file_upload("files/pulfa/MC016/c9616.xml", "application/xml") }
+    let(:file) { fixture_file_upload("files/geo_metadata/non-geo-metadata.xml", "application/xml") }
```

Comment:
Just reading through to see where this is headed, I noticed this change looks like it's different from the description above. Should this remain a non-geo metadata file, or should we update the description?

Reply:
The test used `c9616.xml` to test that Figgy doesn't add the `schema` mime type parameter.  to non-FGDC and non-ISO XML files. The file was chosen at a random. As far as I can tell the spec description is still accurate. I suppose we could move that file out of the /geo_metada/ directory to make it less confusing. Or use an another random xml file from fixtures.

Reply:
That's not necessary — just making sure I was missing something (which I was).